src/main/java/com/commercetools/sync/products/utils/ProductUpdateActionUtils.java

```diff
@@ -94,16 +95,16 @@
    *
    * <p>NOTE: Comparison is done against the staged projection of the old product.
    *
-   * @param oldProduct the product which should be updated.
-   * @param newProduct the product draft where we get the new name.
+   * @param oldProduct the ProductProjection which should be updated.
+   * @param newProduct the ProductProjection draft where we get the new name.
```

Comment:
May I consider that we would check all javadocs in this file and adapt their types ?

```suggestion
   * @param newProduct the product draft where we get the new name.
```

Reply:
b83e7a1